x-pack/platform/plugins/shared/cases/server/cases_analytics/tasks/backfill_task/backfill_task_factory.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { Logger } from '@kbn/logging';
+import type { ElasticsearchClient } from '@kbn/core/server';
+import type { RunContext } from '@kbn/task-manager-plugin/server';
+import { BackfillTaskRunner } from './backfill_task_runner';
+
+interface CaseAnalyticsIndexBackfillTaskFactoryParams {
```

Comment:
```suggestion
interface AnalyticsIndexBackfillTaskFactoryParams {
```

I am renaming this in the Synchronization PR.